scss/_layouts_application.scss

```diff
@@ -0,0 +1,110 @@
+@mixin vf-l-application {
+  .l-application {
+    display: grid;
+    grid-template-areas:
+      'nav  navbar navbar'
+      'nav  main   aside'
+      'nav  status aside';
+    grid-template-columns: min-content 1fr min-content;
+    grid-template-rows: min-content 1fr min-content;
+    height: 100vh;
+    width: 100vw;
+  }
+
+  // Top navigation bar to toggle side navigation on small screens
+  .l-navigation-bar {
+    background: #111;
+    color: #fff;
+    grid-area: navbar;
+    padding: 5px;
```

Comment:
@bartaz Shouldn't styling that won't be part of the final layout should be on a separate class like .l--demo for example?

Reply:
Ah, good catch, I removed it from other panels but not this one.